src/resources.py

```diff
@@ -236,16 +237,26 @@ def on_submit_button_clicked(self):
             }
 
             status, res = api.post(resources_data)
+
+            # response handle
             if not status == 200:
                 # log(neat error message of some kind?)
-                # And probably notify the user?
-                pass
+                if status == 500 and isinstance(res, dict) and "error" in res and "errmsg" in res["error"]:
+                    errmsg = res["error"]["errmsg"]
+                    if "E11000 duplicate key error collection" in errmsg:
+                        msg = "Such IP address is already in use!"
```

Comment:
Sorry to be pedantic, but can you change this to "This IP address is already in use!"